testing/python/fixtures.py

```diff
@@ -4187,3 +4187,21 @@ def test_check_fixture_instantiations():
     )
     result = testdir.runpytest()
     result.assert_outcomes(passed=7)
+
+
+def test_fixture_parametrization_nparray(testdir):
+    testdir.makepyfile(
```

Comment:
I'm not sure if `numpy` is installed in all test environments; if not, I suggest using `importorskip` here:

```suggestion
    pytest.importorskip('numpy')
    testdir.makepyfile(
```

Reply:
Well, I also cannot tell. In this case what should we do?

Reply:
Adding `numpy` to `extras_require` in `setup.py` don't make sure the environments have numpy installed?

Reply:
I'm just commenting in case we get a failure on CI because of this. If everything passes, please disregard this. 👍

Reply:
I see. All tests seem to have passed, the only failed check being something related to coverage - but I didn't understand the reason why.